templates/zerver/emails/email.css

```diff
@@ -84,65 +86,52 @@ table td {
     margin-top: 20px;
 }
 
-.footer td,
-.footer span {
+.content-block,
+.apple-link {
     color: #5f5ec7;
     font-size: 12px;
     text-align: center;
 }
 
-h1,
```

Comment:
We do not use h* tags in our emails. While they may appear in the `missed_message` content, we do not apply styles to them in that context.

Reply:
With those styles applied (after the migration):
<img width="597" alt="image" src="https://user-images.githubusercontent.com/53193850/234189883-d86a064a-b000-486a-93f7-9270672296bb.png">

Reply:
No styles applied (before the migration):
<img width="606" alt="image" src="https://user-images.githubusercontent.com/53193850/234190027-d6341d7d-5723-4c6a-91c4-664426d310c3.png">

Reply:
Yeah this is probably better without the styles.